ooni/errors.py

```diff
@@ -26,6 +27,8 @@
     (CancelledError, 'task_timed_out'),
     (gaierror, 'address_family_not_supported_error'),
     (DNSLookupError, 'dns_lookup_error'),
+    (DNSNameError, 'dns_name_error'),
+    (DNSServerError, 'dns_server_failure'),
```

Comment:
(I suppose there is a reason here why DnsServerError maps to dns_server_failure as opposed to dns_server_error)